server/src/test/java/io/crate/expression/scalar/arithmetic/SignFunctionTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.arithmetic;
+
+import static io.crate.testing.Asserts.isFunction;
+import static io.crate.testing.Asserts.isLiteral;
+
+import org.junit.Test;
+
+import io.crate.exceptions.ConversionException;
+import io.crate.expression.scalar.ScalarTestCase;
+
+public class SignFunctionTest extends ScalarTestCase {
+
+    @Test
+    public void testSign() throws Exception {
+        assertEvaluate("sign(-2)", -1);
+        assertEvaluate("sign(-2.0)", -1.0);
+        assertEvaluate("sign(cast(-2 as bigint))", -1L);
+        assertEvaluate("sign(cast(-2.0 as float))", -1.0f);
+
+        assertEvaluate("sign(11)", 1);
+        assertEvaluate("sign(11.0)", 1.0);
+        assertEvaluate("sign(cast(11 as bigint))", 1L);
+        assertEvaluate("sign(cast(11.0 as float))", 1.0f);
+
+        assertEvaluate("sign(0)", 0);
+        assertEvaluateNull("sign(null)");
+    }
+
+    @Test
+    public void testWrongType() throws Exception {
+        expectedException.expect(ConversionException.class);
```

Comment:
Please use `assertThatThrownBy()`, see for example:
https://github.com/crate/crate/blob/master/server/src/test/java/io/crate/expression/scalar/cast/CastFunctionTest.java#L327

Reply:
done